### PR TITLE
Feat(oidc):oidc grantType add local phone

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Caches/SyncCache.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ internal async Task SyncApiScopeCacheAsync(int id)
     internal async Task SyncIdentityResourceCacheAsync(params int[] ids)
     {
         var identityResources = await IdentityResourceQuery().Where(idrs => ids.Contains(idrs.Id)).ToListAsync();
-        if (identityResources.Count < 0) return;
```
Comment (zhenlei520) [inline src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Caches/SyncCache.cs:40]: format code

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

**Review by zhenlei520 — APPROVED** [review]
@doddgu LGTM

**Review by zhenlei520 — CHANGES_REQUESTED** [review]
@15168440402 Code coverage is too low

**Review by doddgu — APPROVED** [review]
Emm...

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**